### PR TITLE
achi, topic: refactor code, rename func

### PR DESCRIPTION
**Pull request by JeongGoEun**
achi, topic: refactor code, rename func

### CODE REVIEW

File: src/components/Achievement.js [hunk 1]
```diff
@@ -110,6 +101,7 @@ class Achievement extends React.Component {
         break;
       default: break;
     }
+    console.log(this.data.newAchievementItem);
```
Comment (hexoul) [inline src/components/Achievement.js]: delete this debugging line

File: src/components/Achievement.js [hunk 2]
```diff
@@ -131,6 +119,18 @@ class Achievement extends React.Component {
     this[action](targetKey);
   }
 
+  onTabsClick = (value) => {
+    for (var i=0; i < this.data.panes.length; i++) {
+      if (this.data.panes[i].title === topicListArr[value].title) {
+        message.error('Selected duplicate topic.');
+        return;
+      }
+    }
```
Comment (hexoul) [inline src/components/Achievement.js]: Use filter to validate like `onSearch`

File: src/components/Achievement.js [hunk 3]
```diff
@@ -75,20 +79,7 @@ class Achievement extends React.Component {
...
-      }
-    }
-    this.data.panes[this.data.activeKey].title = this.data.originClaimTopics[value].props.children;
-    this.data.newAchievementItem.topic.push({value: topicListArr[value]});
-    
-    this.setState({ isTabChange: true });
-  }
-
-  handleInputChange = (e) => {
+  updateAchieveNewInfo = (e) => {
```
Comment (hexoul) [inline src/components/Achievement.js]: how about changing `updateAchieveNewInfo` to `updateNewAchieveInfo`

File: src/components/Topic.js [hunk 4]
```diff
@@ -49,12 +49,12 @@ class Topic extends React.Component {
         break;
       case 'Pre-fixed':
         this.data.originItems.forEach(element => {
-          if (Object.values(element)[1] < 1025) sortData.push(element);
+          if (Object.values(element)[4] < 1025) sortData.push(element);
```
Comment (hexoul) [inline src/components/Topic.js]: instead of index(number), use key name

File: src/components/Achievement.js [hunk 5]
```diff
@@ -304,7 +304,7 @@ class Achievement extends React.Component {
             onClick={() => this.setState({ addModalVisible: true })}>Add New Achievement</Button>
           <Input.Search
             placeholder='Search by Creator, No., Keyword'
-            onChange={this.onSearchInputChange}
+            onChange={ (e) => this.onSearch(e.target.value) }
```
Comment (hexoul) [inline src/components/Achievement.js]: `(e)` => `e`

File: src/components/Topic.js (hunk 4 above)
Comment (JeongGoEun) [reply src/components/Topic.js]: data structure is array.
For example 
["0xD351858Dd581c4046693cEAe54C169C9f402E16D", "nickname", "nickname", "2018 / Oct / 30", 1026]
So can't use key name.

**Pull request closed without merging**